pkg/operator/ceph/object/bucket/provisioner.go

```diff
@@ -105,6 +105,13 @@ func (p Provisioner) Provision(options *apibkt.BucketOptions) (*bktv1alpha1.Obje
 	}
 	logger.Infof("set user %q bucket max to %d", p.cephUserName, maxBuckets)
 
+	// setting quota limit if it is enabled
```

Comment:
this comment pertains to code above (not changed in this commit):
Why is maxBuckets a `const`? I thought the max number of buckets could be defined in the storage class (or at least in the OBC per above sample yaml).

Reply:
@jeffvance OBC maps to single bucket right? may that option is useful if we are defining quota for cephObjectUser.

Reply:
There is a 1-1 mapping between OBCs and OBs. However, in brownfield cases, 2 OBCs could reference the same storage class and thus the same (existing) bucket.

Reply:
> There is a 1-1 mapping between OBCs and OBs. However, in brownfield cases, 2 OBCs could reference the same storage class and thus the same (existing) bucket.

Yup. What I tried to say there won't multiple buckets for a obc right, the user need to access that bucket and should not create new buckets. So ihmo bucket quota is no much relevant for obc

Reply:
> there won't multiple buckets for a obc
Correct.